src/main/java/unrc/dose/App.java

```diff
@@ -28,12 +34,37 @@ public static void main( String[] args ) {
               if (!Base.hasConnection()) {
                   Base.open();
                 }
+								String url = request.pathInfo();
+								if(url.length() == 12){
+									urlCreate = request.pathInfo().substring(0, 12);
+								}else{
+									if(url.length() == 19){
+										urlReset = request.pathInfo().substring(0, 19);
+									}else{
+										urlCreate = "";
+										urlReset = "";
+									}
+								}
+		            if(!(urlCreate.equals(RUTACREATE))){
+									if(!(urlReset.equals(RUTARESET))){
```

Comment:
Not sure why you check the length of the route first. 

I think that you could directly ask if the route path are contained in the path, something like the following maybe?

```
if (!request.pathInfo().contains(RESET_ROUTE) || !request.pathInfo().contains(CREATE_ACCOUNT_ROUTE))
```

Reply:
probe esta forma de hacerlo y no funcionaba. asi que por ahora lo mantengo asi hasta que se encuentre una forma mejor y mas prolija de hacerlo